server.js

```diff
@@ -6,24 +6,17 @@ const express = require("express");
 const mongoose = require("mongoose");
 
 
-// setup database connection based on current NODE_ENV variable
 if (process.env.NODE_ENV === "production") {
-    if (process.env.MONGO_DB_URL) { // use the MONGO_DB_URL environemnt variable if available:
-        mongoose.connect(`mongodb+srv://${process.env.NAME}:${process.env.PASSWORD}@${process.env.MONGO_DB_URL}`, { useNewUrlParser: true, useUnifiedTopology: true });
-    } else {
-        // otherwise use the original production heroku server:
-        mongoose.connect(`mongodb+srv://${process.env.NAME}:${process.env.PASSWORD}@cluster0.jfqke.mongodb.net/myFirstDatabase?retryWrites=true&w=majority`, { useNewUrlParser: true, useUnifiedTopology: true });
-    }
+    mongoose.connect(`${process.env.MONGO_DB_URL}`, { useNewUrlParser: true, useUnifiedTopology: true });
 
 } else if (process.env.NODE_ENV === 'development') {
-
-    /* ---- For local testing: ----- */
-    mongoose.connect('mongodb://localhost/test?retryWrites=true&w=majority', { useNewUrlParser: true, useUnifiedTopology: true });
+    mongoose.connect('mongodb://localhost/test?retryWrites=true&w=majority', { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true });
```

Comment:
Re-added useCreateIndex: true, which is no longer supported. Other than that looks good!